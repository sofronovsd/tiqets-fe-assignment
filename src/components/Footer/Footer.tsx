import React from 'react';
import styled from "styled-components";

const StyledFooter = styled.footer`
  display: flex;
  padding: 32px 16px;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  gap: 16px;
  align-self: stretch;
  background: #333;
`

const Text = styled.span`
  color: #FFF;
  text-align: center;
  font-family: Roboto;
  font-size: 12px;
  font-style: normal;
  font-weight: 400;
  line-height: 16px; /* 133.333% */
  letter-spacing: 0.6px;
`

const Footer = () => {
  return (
    <StyledFooter>
      <Text>Â© 2014-2021 Tiqets Amsterdam</Text>
    </StyledFooter>
  );
};

export default Footer;
