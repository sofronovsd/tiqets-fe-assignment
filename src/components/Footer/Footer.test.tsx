import {render} from "@testing-library/react";
import Footer from "./Footer";

describe('Footer', () => {
  test('render', () => {
    const { container} = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
