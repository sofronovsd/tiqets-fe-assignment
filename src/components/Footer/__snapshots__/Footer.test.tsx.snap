// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render 1`] = `
<div>
  <footer
    class="sc-bdfCDU kWSemn"
  >
    <span
      class="sc-gsTDqH eioFxa"
    >
      Â© 2014-2021 Tiqets Amsterdam
    </span>
  </footer>
</div>
`;
